### PR TITLE
feature/scss linting

### PR DESCRIPTION
**Pull request by mahboob-awan**

### CODE REVIEW

File: webpack/.stylelintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mahboob-awan) [inline webpack/.stylelintrc]: there are 100s of rules are [available](http://stylelint.io/user-guide/rules/).

I will add them in separate commit with fixes around.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32822

File: package.json [hunk 2]
```diff
@@ -5,11 +5,13 @@
...
     "test:ci:unit": "./node_modules/karma/bin/karma start --browsers PhantomJS --single-run; npm run test:ci:lint",
-    "test:ci:functional": "node ./nightwatch.js -c ./nightwatch.json -e production",
-    "test:ci:lint": "eslint ./src/**/*.js",
     "test:dev:unit": "./node_modules/karma/bin/karma start",
+    "test:ci:functional": "node ./nightwatch.js -c ./nightwatch.json -e production",
     "test:dev:functional": "node ./nightwatch.js -c ./nightwatch.json",
+    "test:ci:lint": "eslint ./src/**/*.js",
     "test:dev:lint": "eslint ./src/scripts/**/*.js",
+    "test:ci:stylelint": "stylelint './src/**/*.scss' --config webpack/.stylelintrc",
+    "test:dev:stylelint": "stylelint './src/**/*.scss' --config webpack/.stylelintrc",
```
Comment (sabeurthabti) [inline package.json]: what is the difference between dev and ci ?

Comment (mahboob-awan) [inline package.json]: Good question :+1: 

@mmahalwy i added "ci" and "dev" entries based on existing ""test:dev:lint" & "test:ci:lint". do we really need both separately?

Comment (sabeurthabti) [inline package.json]: for something it makes sense but in this case they're similar; Just have `test:stylelint`.

Comment (mahboob-awan) [inline package.json]: Fixed. Thanks :)

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32823

**Comment by sabeurthabti** [comment]
Can you [squash](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) your commits and rebase from master?

**Comment by mahboob-awan** [comment]
@sabeurthabti i tried but messed things locally, so better to move :-)

**Comment by ahmedre** [comment]
went ahead and merged this - @mmahalwy check and see if there are any changes you want retroactively made

**Pull request merged**